R/exploratoryfactoranalysis.R

```diff
@@ -134,7 +129,8 @@ ExploratoryFactorAnalysis <- function(jaspResults, dataset, options, ...) {
 
   if (inherits(efaResult, "try-error")) {
     errmsg <- gettextf("Estimation failed. \nInternal error message: %s", attr(efaResult, "condition")$message)
-    modelContainer$setError(.decodeVarsInMessage(names(dataset), errmsg))
+    modelContainer$setError(errmsg)
+    # modelContainer$setError(.decodeVarsInMessage(names(dataset), errmsg))
```

Comment:
@JorisGoosen so there used to be some decoding going on here so that the error message would show decoded column names. I thought this is now decoded automagically, but I'm not 100% sure. Do you know this?

Reply:
definition of `.decodeVarsInMessage` is here btw: https://github.com/jasp-stats/jaspSem/blob/54949a4fee878771854aecbdbf1637a732fc7e8d/R/common.R#L92-L99

Reply:
So @JorisGoosen confirmed this over slack